Tutorials/whatiswot/7-JSON_Schema_in_Practice_2/7-JSON-Schema-Demo.vtt

```diff
@@ -0,0 +1,56 @@
+WEBVTT
+
+00:00:00.000 --> 00:00:06.575 
+In the previous video, we talked about JSON Schema Basics, in this
+
+00:00:06.575 --> 00:00:13.150 
+video, we will continue with generic keywords, Schema composition, and Schema Conditionality.
+
+00:00:13.150 --> 00:00:20.500 
+Some of the annotation keywords used in JSON Schema are title, description, and default.
+
+00:00:20.500 --> 00:00:27.850 
+None of these annotation words are required but enhance good practice for self-documenting Schemas.
+
+00:00:27.850 --> 00:00:32.475 
+The title and description keywords must be strings.
+
+00:00:32.475 --> 00:00:37.737 
+The enum keyword is used to restrict a value to a fixed set of values.
+
+00:00:37.737 --> 00:00:43.000 
+It must be an array with at least one element, where each element is unique.
+
+00:00:43.000 --> 00:00:51.437 
+The const keyword is used to restrict a value to a single value.
+
+00:00:51.437 --> 00:01:03.159 
+JSON Schema includes a few keywords for combining Schemas together, the most used ones are allOf, anyOf, oneOf, and not.
```

Comment:
`not` keyword is not in the slide. Should we add it?

Reply:
I added it. Could you review it?

Reply:
Just noticed that the next sentence says that all the keywords must be assigned to an array but `not` can be assigned to just one schema (i.e. object )

Reply:
Sentence is removed and video is updated.